UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSupplies.json

```diff
@@ -0,0 +1,361 @@
+      "valueCanonical": "http://hl7.org/fhir/us/davinci-dtr/Library/UrologicalSupplies-prepopulation"
+    },
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/cqf-library",
+      "valueCanonical": "http://hl7.org/fhir/us/davinci-dtr/Library/BasicPatientInfo-prepopulation"
+    },
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/cqf-library",
+      "valueCanonical": "http://hl7.org/fhir/us/davinci-dtr/Library/BasicPractitionerInfo-prepopulation"
+    }
+  ],
+  "title": "UrologicalSupplies Order Questionnaire",
+  "status": "draft",
+  "date": "2020-06-22",
+  "publisher": "Da Vinci DTR",
+  "item": [
+    {
+      "linkId": "1",
+      "extension": [
+        {
+          "url": "http://hl7.org/fhir/StructureDefinition/sub-questionnaire",
+          "valueCanonical": "questionnaire/patient-info"
+        }
+      ],
+      "type": "display"
+    },
+    {
+      "linkId": "2",
+      "type": "group",
+      "required": false,
+      "text": "Diagnosis",
+      "item": [
+        {
+          "linkId": "2.1",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"UrologicalSuppliesPrepopulation\".RelavantDiagnosis"
+              }
+            }
+          ],
+          "text": "Patient Qualifying Diagnoses",
+          "type": "open-choice",
+          "repeats": true,
+          "required": true,
+          "answerOption": []
+        },
+        {
+          "linkId": "2.2",
+          "text": "Other Diagnoses",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"UrologicalSuppliesPrepopulation\".CurrentDiagnoses"
+              }
+            }
+          ],
+          "type": "open-choice",
+          "require": true,
+          "repeats": true,
+          "answerOption": []
+        }
+      ]
+    },
+    {
+      "linkId": "3",
+      "type": "group",
+      "required": false,
+      "text": "Order Start Date",
+      "item": [
+        {
+          "type": "date",
+          "required": false,
+          "linkId": "3.1",
+          "text": "Order start date, if different from date of order"
+        }
+      ]
+    },
+    {
+      "linkId": "4",
+      "required": false,
+      "text": "Order Type",
+      "type": "open-choice",
+      "item": [
+        {
+          "linkId": "4.1",
+          "text": "Type of Order",
+          "type": "open-choice",
+          "answerValueSet": "http://hl7.org/fhir/ValueSet/request-intent"
+        }
+      ]
+    },
+    {
+      "linkId": "5",
+      "type": "group",
+      "required": false,
+      "text": "Order",
+      "item": [
+        {
+          "required": false,
```

Comment:
Need to add a text box to pre-populate the "DeviceRequested" from the DeviceRequest. And probably change the label to "other orders: (category)" for those orders populated from valueset? We don't have the ability to merge main order and supply yet and I am fine to leave the order selection out.